test/e2e/scheduling/opaque_resource.go

```diff
@@ -1,305 +0,0 @@
-/*
```

Comment:
Can't this file be repurposed for testing extended resources?

Reply:
These tests always had flake problems and therefore are disabled by default. I agree we should have e2e tests for extended resources but given the state of these tests I’d prefer to add them back in another PR.